guides/common/modules/proc_configuring-project-server-for-cloud-connection.adoc

```diff
@@ -0,0 +1,22 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-{project-context}-server-for-cloud-connection"]
+= Configuring {ProjectServer} for cloud connection
+
+[role="_abstract"]
+You can configure {ProjectServer} to connect to the {RHCloud}.
+By performing this action, you run a remote job on your {ProjectServer} to enable the connection.
```

Comment:
```suggestion
You can configure {ProjectServer} to connect to the {RHCloud} by using remote execution.
```

I do not fully understand line 8. Does this mean run self-REX? If so, is this necessary in the abstract?

Reply:
Permission added. Intro reworded.